templates/base.html

```diff
@@ -47,102 +48,114 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
-        <div class="off-canvas-wrapper">
-            <div class="off-canvas-wrapper-inner" data-off-canvas-wrapper>
-                <div class="off-canvas position-right" id="offCanvas" data-off-canvas data-position="right">
-                    <!-- Close button -->
-                    <button class="close-button" aria-label="Close menu" type="button" data-close>
-                        <span aria-hidden="true">&times;</span>
-                    </button>
+        {% if user.is_authenticated %}
```

Comment:
Tried doing this but couldn't get the if to work correctly. How would I go about minimizing the code?